stock_location_tray/tests/test_stock_move_line.py

```diff
@@ -27,16 +27,16 @@ def create_stock_move(self):
             {
                 "move_id": move.id,
                 "product_id": move.product_id.id,
-                "qty_done": 1,
+                "quantity": 1,
                 "product_uom_id": move.product_uom.id,
                 "location_id": move.location_id.id,
                 "location_dest_id": move.location_dest_id.id,
             }
         )
-        self.move = move
+        return move
 
     def test_compute_tray_matrix(self):
-        self.create_stock_move()
+        self.move = self.create_stock_move()
```

Comment:
You're on the right path here.
But we should also try to avoid setting `self.move` here, too

Instead use a local variable if possible

`move = self.create_stock_move()`